xline-client/src/error.rs

```diff
@@ -1,9 +1,9 @@
 use thiserror::Error;
 
-/// Xline client result
+/// The result type for `xline-client`
 pub type Result<T> = std::result::Result<T, ClientError>;
 
-/// Client Error
+/// The Error type for `xline-client`
 #[derive(Error, Debug)]
 #[non_exhaustive]
 pub enum ClientError {
```

Comment:
Some comments on the specific error types are capitalized, like `Rpc Error`,  while the others are lowercase, like `Propose error`. Please use lowercase consistently, say `rpc error`

Reply:
done